ex1/functions.py

```diff
@@ -47,15 +47,94 @@ def f():
 
 # Define a function named add that takes two numbers and returns the sum.
 
+def add(a,b):
+    """ Adds two numbers
+	
+    Parameters
+    ----------
+    a,b : int,float
+		
+    Returns
+    -------
+    Int, float
+		a+b
+    """
+    return a+b
+    
 # Define a function named to_tuple that takes three arguments and returns a
 # tuple of these three arguments.
 
+def to_tuple(a,b,c):
+    """ Creates a tuple out of 3 variables
+	
+    Parameters
+    ----------
+    a,b,c: any type
+	
+    Returns
+    -------
+    tuple
+		(a,b,c)
+    """
+    return (a,b,c)
```

Comment:
You do not need the brackets here since multiple return values are automatically packed into a tuple.